src/components/Badge/Badge.mdx

```diff
@@ -11,33 +16,35 @@ Badges are a flexible component used to display small nuggets of text prominentl
 
 ## How to Use
 
-There are 6 variants of badges: info, success, warning, error, turquoise, and flint. By default, the badge will display as info.
+There are 6 variants of badges: default, info, success, warning, error, and gradient.
```

Comment:
@shannamurry The `turquoise` variant is used actively in some places. I know in campaigns for sure like over [here](https://github.com/lob/dashboard-vue/blob/ab00e96874370cc1a14442987e819708aa8377ce/src/utils/campaign.js#L21-L35) when displaying the Status Section in campaign detail view, and the badge in the live campaigns table.

So should we change that now before this PR? Or leave `turquoise`, make necessary updates to the dashboard-repo, and then come back and clean this up?

What would happen if this is deployed, but the dashboard still uses `variant = turquoise`?

Reply:
@siddharthpant92 this was the plan! any turquoise/flint variants will be changed to the 'default' variant when we update the dash to the v of components with the new Badge